### DISCUSSION TITLE
Developer Installation

### DISCUSSION THREAD

**Discussion by puttygutt** [bookwyrm-social/bookwyrm#3176] [General]
Hello,

Newbee here.  I'm in the process of setting up the dev environment on a DigitalOcean Ubuntu droplet.  I'm running into an issue when I run the ./bw-dev up command.  I'm getting the error: /bin/sh: 1: npm: not found at the docker startup.  npm is installed on the Ubuntu environment.  Before messing with the script, I'd like to know if I need to include a setting in one of the config files.  Please advise.  

Thanks, 

Peter

**Comment by dato** [comment]
Hello!

Yes, something in the Python Docker images changed that makes the script fail. It is reported in #3173, and there is a fix (if you want to check it out and apply it) in #3175. (It’s a simple change in Dockerfile, and an extra file called nodejs.pref.)

**Comment by puttygutt** [comment]
Thank you.  Your guidance solved the start-up issue.
